### PR TITLE
Fix a string split issue on powershell 5.1

### PR DESCRIPTION
**Pull request by patrick330602**
This should help with issue #17

### CODE REVIEW

**Comment by patrick330602** [comment]
Well, being too exited when I find the solution... I just fixed 2 issues on both powershell 5.x and 6 so that all of them works now 😄 
it looks like this:
image

**Comment by tylerl0706** [comment]
Nice! Although, it looks like the `(Default)` is still showing up in the Distribution name.

The split you do on line 165 is supposed to remove that from going into the created object.

**Comment by patrick330602** [comment]
Yeah, I notice that, Just fixed it. I just hate the fact that there exist two `split` for string which have a totally different behaviours.
image

File: PSWsl.psm1 [hunk 1]
```diff
@@ -157,12 +157,12 @@ function Get-WslDistribution {
     
     begin {
         # parse
-        $array = [System.Text.Encoding]::Unicode.GetString([System.Text.Encoding]::UTF8.GetBytes((wslconfig.exe /l))).Split("   ", [System.StringSplitOptions]::RemoveEmptyEntries) | Select-Object -Skip 1 -Unique
-
+        $array = [System.Text.Encoding]::Unicode.GetString([System.Text.Encoding]::UTF8.GetBytes((wslconfig.exe /l))) -split '\s\s+' | Select-Object -Skip 1 -Unique
+        $array = $array[0..($array.length-2)]
```
Comment (tylerl0706) [inline PSWsl.psm1]: Just curious... Why -2? I would expect -1.

Could you add a comment here?

Comment (patrick330602) [reply PSWsl.psm1]: This is to trim the extra item in the array, I do not know why but there is a extra item inside `$array` after line 160, so line 161 is basically removing that item at the end from the `$array`.

Comment (tylerl0706) [reply PSWsl.psm1]: I see. The extra item must be an empty space.

When you say you tried Pipeline, do you mean:

```
Get-WSLDistribution
```

Piping into something? Or piping something into it?

Comment (patrick330602) [reply PSWsl.psm1]: Well... I mean using pipeline on line 160 so that line 161 can be removed, the pipeline for `Get-WSLDistribution` works anyway

Comment (tylerl0706) [reply PSWsl.psm1]: No worries :) how it is now is good!

**Comment by tylerl0706** [comment]
Merging this :)

**Comment by tylerl0706** [comment]
I'll throw a release out soon-ish

**Comment by tylerl0706** [comment]
Released! https://www.powershellgallery.com/packages/PSWsl/0.2.3 🎉 awesome work

**Pull request merged**